pkg/controllers/managedserviceaccount/log_serviceaccount_controller.go

```diff
@@ -0,0 +1,133 @@
+package managedserviceaccount
```

Comment:
why does this controller belong to the ocm controller instead of the ocm proxyserver?

Reply:
The log permission rbac is managed by the ocm-controller, so I think it makes sense to create the managed SA in the ocm-controller.